### PR TITLE
DOCS-7102 fixed broken links

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/pages/oauth-grant-types-about.adoc [hunk 1]
```diff
@@ -3,14 +3,14 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-OAuth 2.0 specify the following methods (ie Grant Types) for requesting for a token.
+OAuth 2.0 provides the following grant types methods for requesting for a token:
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/oauth-grant-types-about.adoc]: "specifies" is probably the right word. OAuth 2.0 is just a spec and every software vendor implements it.

File: modules/ROOT/pages/oauth-grant-types-about.adoc [hunk 2]
```diff
@@ -3,14 +3,14 @@ ifndef::env-site,env-github[]
...
 
 * AUTHORIZATION_CODE
 * IMPLICIT
 * RESOURCE_OWNER_PASSWORD_CREDENTIALS
 * CLIENT_CREDENTIALS
 
-For those RAML-based APIs in which the RAML should be updated with the OAuth 2.0 security schema, the following table maps the RAML grant types to the grant type names in the OAuth 2.0 policy configuration.
+For the RAML-based APIs in which the RAML must be updated with the OAuth 2.0 security schema, the following table maps the RAML grant types to the grant type names in the OAuth 2.0 policy configuration:
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/oauth-grant-types-about.adoc]: typically there wouldn't be an article here at all, just "For RAML-based APIs in which the RAML..."
What does the reader have to do? Replace the RAML grant type with the corresponding OAuth 2.0 grant type? It's great to have the mapping, but I'm not sure what the task is (and therefore, why the info is here). if there's any way to break up this long sentence into two smaller ones, that'd be great. I couldn't try as I don't know what we're telling the reader to do here.

File: modules/ROOT/nav.adoc [hunk 3]
```diff
@@ -155,12 +155,12 @@
  ** xref:analytics-event-api.adoc[Reviewing Analytics Event API]
  ** xref:analytics-event-forward.adoc[Reviewing Analytics Event Forwarding]
 * xref:mule-oauth-provider-landing-page.adoc[Mule OAuth 2.0 Provider]
- ** xref:oauth2-provider-configuration.adoc[Mule OAuth 2.0 Provider Configuration]
+ //** xref:oauth2-provider-configuration.adoc[Mule OAuth 2.0 Provider Configuration]
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/nav.adoc:158]: Can we delete instead of comment out? To keep debt from accumulating in the file.

**Review by Mulesoft-Mysti — APPROVED** [review]
just a few comments.

File: modules/ROOT/pages/oauth-grant-types-about.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/oauth-grant-types-about.adoc]: Done! Thanks!

File: modules/ROOT/pages/oauth-grant-types-about.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/oauth-grant-types-about.adoc]: Done! Thanks!

File: modules/ROOT/nav.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/nav.adoc:158]: Yes, will do once I get to know from the Engineer what is he trying to do by removing these files. I'll go ahead and merge the content though.

**Comment by fermujica** [comment]
Commenting out the links does not fix the broken links. You need to remove broken references.
See https://github.com/mulesoft/docs-api-manager/pull/194

**Pull request merged**